streams/br.m3u

```diff
@@ -24,6 +24,10 @@ https://stream01.msolutionbrasil.com.br/hls/canal25/live.m3u8
 https://arkyvbre1g.zoeweb.tv/fiocruz/fiocruz/playlist.m3u8
 #EXTINF:-1 tvg-id="CartoonNetworkBrazil.br" status="online",Cartoon Network (Spanish)
 https://playout.cdn.cartoonnetwork.com.br/playout_02/playlist.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkBrazil.br" status="online",Cartoon Network (Portuguese)
+https://playout.cdn.cartoonnetwork.com.br/playout_01/playlist.m3u8
+#EXTINF:-1 tvg-id="CartoonitoBrazil.br" status="online",Cartoonito (Portuguese)
+https://playout.cdn.cartoonnetwork.com.br/playout_03/playlist.m3u8
```

Comment:
```suggestion
https://playout.cdn.cartoonnetwork.com.br/playout_03/playlist.m3u8
#EXTINF:-1 tvg-id="" status="online",Cartoonito (Spanish)
https://playout.cdn.cartoonnetwork.com.br/playout_04/playlist.m3u8
```

Reply:
From data I've seen, it seems it is Cartoonito Latinoamérica. Corresponds to the Cartoon Network version, having a "LA" in one of it's shows, LA being LatinoAmérica.